### PR TITLE
[MORPHY] disabling cloud volume delete functionallity for autosde volumes

### PR DESCRIPTION
**Pull request by agrare**
Morphy backport of https://github.com/ManageIQ/manageiq-providers-autosde/pull/134

### CODE REVIEW

**Review by llivne — APPROVED** [review]

**Comment by Fryguy** [comment]
@llivne In the future, please don't merge non-master PRs without it going through the release manager (which is me), as I have to do extra issue management with those.

**Pull request merged**
